src/adapters/pagarme/risk-analysis.js

```diff
@@ -3,18 +3,24 @@ const {
   always,
   applySpec,
   assoc,
-  ifElse,
-  propEq,
-  pipe,
+  cond,
   of,
+  pathEq,
+  pipe,
+  prop,
+  T,
 } = require('ramda')
 
-const isIndividual = propEq('document_type', 'cpf')
+const isIndividual = pathEq(['recipient', 'document_type'], 'cpf')
 
-const policyId = ifElse(isIndividual, always(6), always(5))
+const getPolicyId = cond([
+  [prop('policyId'), prop('policyId')],
```

Comment:
Why two `prop('policyId')`?

Reply:
@uselessdev You're right! Shouldn't this be `[has('policyId'), prop('policyId')]`??